src/Hedgehog/Property.fs

```diff
@@ -4,15 +4,15 @@ open System
 
 [<Struct>]
 type Property<'a> =
-    | Property of Gen<Journal * Outcome<'a>>
+    | Property of PropertyConfig * Gen<Journal * Outcome<'a>>
 
 module Property =
 
-    let ofGen (x : Gen<Journal * Outcome<'a>>) : Property<'a> =
-        Property x
+    let ofGen (gen : Gen<Journal * Outcome<'a>>) : Property<'a> =
+        Property (PropertyConfig.defaultConfig, gen)
```

Comment:
I gave the `PropertyConfig.defaultConfig` zero thought.

Reply:
We'll need to audit the use of this function, if it's being used as `prop |> toGen |> doGenThing |> ofGen`, then we'll lose the original configuration.

Reply:
Indeed.  All such calls should be replaced with `Property.Generator.map` (as I structured/named it in another comment).

Reply:
How about, it a PR before this one, we change `Property` from a single case discriminated union to a record with a single field as well as expressing all maps over that field as such?  Then I think it would easier to verify where `PropertyConfig.defaultConfig` should be used.